### PR TITLE
feat: Add support for OpenSuse Aeon

### PR DESCRIPTION
**Pull request by TheMuso**
# Description

This adds support for downloading the latest OpenSuse Aeon iso.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have tested my code in common scenarios and confirmed there are no regressions
- [ ] I have added comments to my code, particularly in hard-to-understand sections

### CODE REVIEW

**Comment by lj3954** [comment]
Chore is used for CI changes. This is a feature addition.

File: quickget [hunk 1]
```diff
@@ -2204,6 +2204,9 @@ function get_opensuse() {
     elif [ "${RELEASE}" == "microos" ]; then
         ISO="openSUSE-MicroOS-DVD-x86_64-Current.iso"
         URL="https://download.opensuse.org/tumbleweed/iso"
+    elif [ "${RELEASE}" == "aeon" ]; then
+        ISO="opensuse-aeon.x86_64.iso"
+        URL="https://mirrorcache-au.opensuse.org/tumbleweed/appliances/iso"
```
Comment (philclifford) [inline quickget:2209]:
```suggestion
        URL="https://mirrorcache.opensuse.org/tumbleweed/appliances/iso"
```
Shouldn't this be the URL to enable proper redirection?

Comment (TheMuso) [reply quickget:2209]: Whoops, yes it should, copy paste error on my part. Fixing.

**Comment by TheMuso** [comment]
As for the prefix, I looked at the git log for quickget, and saw chore being used, not looking closely as to what was being changed, so thanks for changing the commit title and PR title, will keep in mind for next time.

**Review by flexiondotorg — APPROVED** [review]

**Pull request merged**